source/crud/read-operations/cursor.txt

```diff
@@ -0,0 +1,150 @@
+=========================
+Access Data From a Cursor
+=========================
+
+.. default-domain:: mongodb
+
+Read operations that return multiple documents do not immediately return
+all values matching the query. Because a query can potentially match
+very large sets of documents, these operations rely upon an
+object called a **cursor** that fetches documents in batches to reduce
+both memory consumption and network bandwidth usage. Cursors
+are highly configurable and offer multiple interaction paradigms for
+different use cases.
+
+The following functions directly return cursors:
+
+- ``Collection.find()``
+
+- ``Collection.aggregate()``
+
+- ``Collection.listIndexes()``
+
+- ``Db.aggregate()``
+
+- ``Db.listCollections()``
+
+Other methods, like ``Collection.findOne()`` and ``Collection.watch()``
+use cursors to generate results, but do not directly expose those
+cursors to the method caller.
+
+.. warning::
+
+   Do not combine different cusor paradigms on a single cursor.
```

Comment:
This term also bothered me, but I can't think of a better term to explain this. I'm going to bring up paradigms explicitly below here and move this warning down so that this isn't the first/only time readers hear the phrase "cursor paradigm."